specification/cognitiveservices/data-plane/CustomVision/Prediction/stable/v3.0/Prediction.json

```diff
@@ -0,0 +1,917 @@
+﻿{
+  "swagger": "2.0",
+  "info": {
+    "version": "3.0",
+    "title": "Custom Vision Prediction Client"
+  },
+  "host": "southcentralus.api.cognitive.microsoft.com",
+  "basePath": "/customvision/v3.0/prediction",
+  "schemes": [
+    "https"
+  ],
+  "paths": {
+    "/{projectId}/classify/models/{publishedModelName}/url": {
+      "post": {
+        "tags": [
+          "ImagePredictionApi"
+        ],
+        "summary": "Classify an image url and saves the result.",
+        "operationId": "ClassifyImageUrl",
+        "consumes": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml",
+          "application/x-www-form-urlencoded"
+        ],
+        "produces": [
+          "application/json",
+          "text/json",
+          "application/xml",
+          "text/xml"
+        ],
+        "parameters": [
+          {
+            "name": "projectId",
+            "in": "path",
+            "description": "The project id.",
+            "required": true,
+            "type": "string",
+            "format": "uuid",
+            "x-nullable": false
+          },
+          {
+            "name": "publishedModelName",
+            "in": "path",
+            "description": "Specifies the name of the model to evaluate against.",
+            "required": true,
+            "type": "string",
+            "x-nullable": true
+          },
+          {
+            "name": "imageUrl",
+            "in": "body",
+            "description": "An {Iris.Web.Api.Models.ImageUrl} that contains the url of the image to be evaluated.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ImageUrl"
+            }
+          },
+          {
+            "name": "application",
+            "in": "query",
+            "description": "Optional. Specifies the name of application using the endpoint.",
+            "required": false,
+            "type": "string",
+            "x-nullable": true
```

Comment:
I would strongly recommend revisiting all use of x-nullable. It should only be used in specific cases where there is a difference between not sending a value and sending a null value. Which should be very rare.

Reply:
Revisited and updated. It's required on almost all value types to not generate a nullable via autorest. Removed some that didn't affect generation.